test/bin/build-icu.sh

```diff
@@ -52,14 +52,18 @@ fail_on_error() {
     case $VARIANT in
     windows-64)
         # The cmd business brings all the needed compiler binaries into the shell's environment variables.
-        cmd /c "C:\Program^ Files^ ^(x86^)\Microsoft^ Visual^ Studio^ 14.0\VC\vcvarsall.bat amd64 && bash -c 'cd $ICU_BUILD_DIR ^&^& eval $CONFIGURE'"
+        #cmd /c "C:\Program^ Files^ ^(x86^)\Microsoft^ Visual^ Studio^ 14.0\VC\vcvarsall.bat amd64 && bash -c 'cd $ICU_BUILD_DIR ^&^& eval $CONFIGURE'"
```

Comment:
Normally, I'd say to delete these comments, but I think this is ok for this repo.

Reply:
Good point!
Cleaned-up 👍